aiosmtpd/tests/test_main.py

```diff
@@ -116,11 +124,11 @@ def test_nosetuid(self):
     def test_debug_0(self):
         # For this test, the runner will have already set the log level so it
         # may not be logging.ERROR.
-        log = logging.getLogger('mail.log')
-        default_level = log.getEffectiveLevel()
-        with patch.object(log, 'info'):
+        _log = logging.getLogger('mail.log')
+        default_level = _log.getEffectiveLevel()
+        with patch.object(_log, 'info'):
             main(('-n',))
-            self.assertEqual(log.getEffectiveLevel(), default_level)
+            self.assertEqual(_log.getEffectiveLevel(), default_level)
```

Comment:
Variable renamed to suppress complaint of shadowing a higher-level `log` variable.